src/content/team/team.json

```
@@ -10,32 +19,104 @@
-        "email": "jeevanantham.a@husky.neu.edu",
-        "linkedIn": "https://www.linkedin.com/in/arunjeevanantham/"
+        "email": "takayama.a@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/alexander-takayama-748909a8/"
       }
     },
     {
       "name": "Sauharda Rajbhandari",
-      "role": "Marketing Director",
+      "role": "Operations Director",
       "profileImage": "./profileImages/sauharda.png",
       "socialMedia": {
         "email": "rajbhandari.sa@husky.neu.edu",
         "linkedIn": "https://www.linkedin.com/in/sauhardarajbhandari/"
       }
     },
+    {
+      "name": "Yash Shetty",
+      "role": "Team Lead - GraduateNU",
+      "profileImage": "./profileImages/yash.png",
+      "socialMedia": {
+        "email": "shetty.y@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/yash-shetty-431996141/"
+      }
+    },
+    {
+      "name": "Will Stenzel",
+      "role": "Team Lead - Khoury Office Hours App",
+      "profileImage": "./profileImages/will.png",
+      "socialMedia": {
+        "email": "stenzel.w@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/company/sandboxnu/"
+      }
+    },
+    {
+      "name": "Mitch Gamburg",
+      "role": "Team Lead - SearchNEU",
+      "profileImage": "./profileImages/mitch.png",
+      "socialMedia": {
+        "email": "gamburg.m@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/mitchell-gamburg-6691a569/"
+      }
+    },
+    {
+      "name": "Aislin Black",
+      "role": "Team Lead - Cheminformatics",
+      "profileImage": "./profileImages/aislin.png",
+      "socialMedia": {
+        "email": "black.a@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/aislin-black-0b6b64180"
+      }
+    },
+    {
+      "name": "Tal Usvyatsky",
+      "role": "Team Lead - PharmD Tracker",
+      "profileImage": "./profileImages/tal.png",
+      "socialMedia": {
+        "email": "usvyatsky.t@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/tal-usvyatsky/"
+      }
+    },
+    {
+      "name": "Jacob Chvatal",
+      "role": "Team Lead - Eliza Project",
+      "profileImage": "./profileImages/jacob.png",
+      "socialMedia": {
+        "email": "chvatal.j@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/jacob-chvatal/"
+      }
+    },
+    {
+      "name": "Cindy Luo",
+      "role": "Team Lead - Visualizer Project",
+      "profileImage": "./profileImages/cindy.png",
+      "socialMedia": {
+        "email": "luo.xiny@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/cindy-luo-85945b176/"
+      }
+    },
+    {
+      "name": "Catherine Titcomb",
+      "role": "Head of Brand",
+      "profileImage": "./profileImages/catherine.png",
+      "socialMedia": {
+        "email": "titcomb.c@husky.neu.edu",
+        "linkedIn": "https://www.linkedin.com/in/catherinetitcomb/"
+      }
+    },
     {
       "name": "Fiona Gridley",
       "role": "Design Lead",
```

```suggestion
      "role": "Designer",
```
I don't think we ever made a Design Lead role